lib/shopify_api/utils/session_utils.rb

```diff
@@ -48,6 +41,25 @@ def current_session_id(auth_header, cookies, online)
           end
         end
 
+        sig do
+          params(
+            id_token: T.nilable(String),
+            online: T::Boolean,
+          ).returns(String)
+        end
+        def session_id_from_shopify_id_token(id_token:, online:)
```

Comment:
I don't like overloading _another_ param into the `current_session_id` method, since it's already expecting input that is either-or, either "auth_token" OR "cookies", it'll just create even more branching for the logic.. 

Perhaps we change the "auth_token" input to BE "shopify_session_id" and just strip "Bearer " before parsing the rest of the token if "Bearer" exists.. that way they can still pass both auth header AND URL into the same method?

Reply:
Thanks! I just updated the PR with the change